commitizen/commands/check.py

```diff
@@ -32,9 +33,12 @@ def __init__(self, config: BaseConfig, arguments: Dict[str, str], cwd=os.getcwd(
         self.cz = factory.commiter_factory(self.config)
 
     def _valid_command_argument(self):
-        if not (
-            bool(self.commit_msg_file) ^ bool(self.commit_msg) ^ bool(self.rev_range)
-        ):
+        sources = (
```

Comment:
The naming `sources` does not look straightforward to me. Currently, I do not have a better name for it. But we could at least add some comments to it.

Reply:
it's the amount/number of params provided, so something like `params_amount` or `number_params_provided` could be